js/scroll.js

```diff
@@ -1,18 +1,47 @@
-const SCROLL_DELAY = 3000;
-var scrollDistance;
-var intervalScroll;
+const SCROLL_PLAN_DELAY = 3000, SCROLL_INFO_DELAY = 3000;
+var intervalScrollPlan, intervalScrollInfo;
+var infoContainer = document.getElementById('info-container');
```

Comment:
Ok so the same applies to the boxContainer on line 1. Should we also call updateVertretungsplan() in jsonparser.js only on load?